src/components/search/ClimbSearchByName.tsx

```diff
@@ -3,47 +3,57 @@ import { useRouter } from 'next/router'
 import { typesenseSearch } from '../../js/typesense/TypesenseClient'
 import { Autocomplete } from './Autocomplete'
 import { SearchByNameTemplate } from './ResultTemplates'
+import { debounced } from '../../js/utils'
 
 export const ClimbSearchByName = ({ isMobile = true, placeholder = 'Try \'Levitation 29\', \'technical crimpy\', or \'Lynn Hill\'' }: {isMobile?: boolean, placeholder?: string}): JSX.Element => {
   const router = useRouter()
+
   useEffect(() => {
```

Comment:
this useEffect is causing the autocomplete to fail in some situations. Example: click on "Climb Search", enter value, get result, change tab to "Places to Climb", change tab back to "Climb Search", autocomplete will not produce results.

Reply:
The focus code is buggy now that there are 2 search boxes (probably due to multiple 'aa-Input' classes).  You can remove it.

Reply:
maybe add an id to each autocomplete

and then `document.getElementById('cragAutocomplete').focus()`

Reply:
You can try to see if it works. I haven't figured out a way to add an ID to the Input element (it's created by the 3rd party Autocomplete component)

Reply:
So <Tabs/> was actually the culprit

the input needs to wait for the tabs to finish switching before selecting the input 

you can see that change here
https://github.com/clintonlunn/open-tacos/pull/3/files

Reply:
I believe the culprit was 
className="ease-in-out duration-500"

from [develop/src/components/search/Tabs.tsx](https://github.com/clintonlunn/open-tacos/blob/develop/src/components/search/Tabs.tsx)

Reply:
@MichaelDimmitt you're right!  I just did a quick check on my local build